code/modules/mob/living/say.dm

```diff
@@ -255,7 +255,10 @@ proc/get_radio_key_from_channel(var/channel)
 		if(M.client)
 			speech_bubble_recipients.Add(M.client)
 	spawn(0)
-		flick_overlay(image('icons/mob/talk.dmi', src, "h[speech_bubble_test]",MOB_LAYER+1), speech_bubble_recipients, 30)
+		if(istype(src.loc, /obj/mecha))

```

Comment:
this is snowflakey and makes me cry.
you should probably have something like
```/atom/proc/speech_bubble(bubble_state = "",bubble_loc = src, list/bubble_recipients = list())```
and something like
```
if(loc && !isturf(loc)) 
    var/atom/A = loc 
    A.speech_bubble(stuff...)
else
    speech_bubble(stuff...)
```